signer/core/cliui.go

```diff
@@ -242,6 +244,24 @@ func (ui *CommandlineUI) OnApprovedTx(tx ethapi.SignTransactionResult) {
 }
 
 func (ui *CommandlineUI) OnSignerStartup(info StartupInfo) {
+	accounts, err := ui.api.ListAccounts(context.Background())
+	if err != nil {
+		fmt.Print("error listing accounts", err)
+	}
+	if len(accounts) == 0 {
+		fmt.Println("No accounts known to Clef")
+	} else {
+		// account info to string for nicer printing
+		var addresses string = "\n"
+		fmt.Println("Accounts known to Clef:")
+		for i, account := range accounts {
+			// concat string to avoid repeating "INFO" on terminal
+			addresses += fmt.Sprintf("Account %v: %s at %s", i, account.Address, account.URL)
+			addresses += "\n"
```

Comment:
Instead of `+= "\n"`, why not add the `\n` to the line above? 
```suggestion
			addresses += fmt.Sprintf("Account %v: %s at %s\n", i, account.Address, account.URL)
```